packages/generator/src/input-path-provider.ts

```diff
@@ -30,9 +30,12 @@ export function inputPaths(
   });
 }
 
-export function swaggerPathForEdmx(edmxPath: PathLike): PathLike {
-  const extension = new RegExp(`${extname(edmxPath.toString())}\$`);
-  return edmxPath.toString().replace(extension, '.json');
+export function swaggerPathForEdmx(edmxPath: PathLike): PathLike | undefined {
```

Comment:
So, this means, only use the json when it exists, which I checked;)

Reply:
Correct! + we also check for .JSON ;)